README.md

```diff
@@ -1,3 +1,12 @@
+# Before Getting Started
+
+Please be sure when running the front end repository that you have the correct versions of npm and node.
+The correct version can be found and installed [here](https://nodejs.org/download/release/v14.17.0/).
+
+To check if your versions are correct you can use the following commands:
+Check node version with `node -v` -> This should show v14.17.0 in the console
+Check npm version with `npm -v` -> This should show 6.14.x in the console
```

Comment:
Why not the latest, stable version of npm?

Reply:
Will swap to latest release of node. Looks like this is version 20.2.0 right now.

Reply:
Updated